index.html

```diff
@@ -5,22 +5,101 @@
 
-		<title>Blank Template</title>
+		<title>Homework Assignment 3</title>
 
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
-
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,800" rel="stylesheet">
 	</head>
 	<body>
-		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1>Relaxr</h1>  
+				<nav class="clearfix"> 
+					<ul class="alignright clearfix"> 
+						<li>	
+							<a href="#">About</a>
+						</li>
+						<li>
+							<a href="#">FAQ</a>
+						</li>
+						<li>
+							<a href="#">Team</a>
+						</li>
+						<li>
+							<a href="#">Contact Us</a>
+						</li>
+						<li>
+							<a href="#">Blog</a> 
+						</li>
+					</ul>	 
+				</nav>
 			</header>
+
+			<div class="container clearfix">
+				<div class="main alignleft">
+					<div class="article">
+						<h3>How I implemented Relaxr in 2 weeks and changed my life</h3>
+
+						<img src="images/blog_photo1.jpg" alt="Rowers">
+						<p>
+						<span class="drop alignleft">L</span>
+						orem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+						tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+						quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+						consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+						cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+						proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+						<div class="readmore">Read more &gt</div>
+						<hr/>
+					</div>
+
+					<div class="article">
+						<h3>I traveled across the globe thanks to Relaxr’s Excel automation</h3>
+
+						<img src="images/blog_photo2.jpg" alt="Boats">
+
+						<p>
+						<span class="drop alignleft">L</span>
+						orem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+						tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+						quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+						consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+						cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+						proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+						<div class="readmore">Read more &gt </div><hr/>
+					</div>
+				</div><!--#main-->
+				
+				<div class="sidebar alignleft">
+					<h4>Categories</h4>
+					<div>Success stories</div>	
+					<div>Stats</div>
+					<div>How tos</div>
+					<div>Best of</div>
+					<div>Business tips</div>
+					<h4>About Relaxr</h4>
+					<p>Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.</p>
+					<div>Learn More</div>
+					<img src="images/ad-unit.png" alt="ad">
```

Comment:
considering adding a class to this like `class="ad"` and then creating some white space between it and the content above with `margin-top: 30px` or something like that.